engines/zvision/graphics/render_manager.cpp

```diff
@@ -380,59 +504,55 @@ void RenderManager::scaleBuffer(const void *src, void *dst, uint32 srcWidth, uin
 	}
 }
 
-void RenderManager::blitSurfaceToSurface(const Graphics::Surface &src, const Common::Rect &_srcRect , Graphics::Surface &dst, int _x, int _y) {
-	Common::Rect srcRect = _srcRect;
+void RenderManager::blitSurfaceToSurface(const Graphics::Surface &src, Common::Rect srcRect, Graphics::Surface &dst, int _x, int _y) {
+	debug(9, "blitSurfaceToSurface");
+	Common::Point dstPos = Common::Point(_x, _y);
+	//Default to using whole source surface
```

Comment:
```suggestion
	// Default to using whole source surface
```
*** Same for all the comments below ***